test/protocol/protocol.test.js

```diff
@@ -626,7 +610,7 @@ describe('Protocol tests', () => {
 
             expect(events).to.be.an('array');
             const bidTakenEvent = events.find(event => event.event === 'BidTaken');
-            console.log(JSON.stringify(events));
```

Comment:
Fuck, I've been looking for that!

Reply:
@kipliklotrika  I am trying to minimize amount to logged lines to keep travis logs readable.
Now we have console.debug which achieves that, so if we ever need to debug test case one just needs to replace all console.debugs with console.logs.
Would that be OK?